db/seeds.rb

```diff
@@ -59,7 +59,7 @@
 end
 
 # reindex all talk in MeiliSearch
-Talk.reindex! unless Rails.env.test? || Rails.env.production?
+#Talk.reindex! unless Rails.env.test? || Rails.env.production?
```

Comment:
When I setup the project, this failed because of the changes of 0b929c72

Reply:
good catch, we are i the middle of upgrading Meilisearch and Kamal 2 this is why we have a fallback to SQLITE search for now. Also for new comers it is simpler if the porject can run without a Meilisearch dependency.